src/helpers/decorators.ts

```diff
@@ -0,0 +1,21 @@
+import { type Purchases } from "../main";
```

Comment:
Only importing the type to avoid a circular reference

Reply:
Nice one, TIL!